src/main/java/isaac/bastion/manager/BastionBlockManager.java

```diff
@@ -143,10 +144,22 @@ public boolean onCooldown(UUID player, BastionType type){
 		}
 		
 		for(Block block: result) {
-			toReturn.addAll(getBlockingBastions(block.getLocation(),accessors));
+			preblocking.addAll(getBlockingBastions(block.getLocation(),accessors));
 		}
-		
-		return toReturn;
+
```

Comment:
I've returned back old method and renamed new method to shouldStopBlockByBlockingBastion